alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +26,16 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec:
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	go get -u github.com/securego/gosec/cmd/gosec
+endif
+	$(GOSEC) \
+		-severity medium -confidence medium \
```

Comment:
those flags need to be configurable and I think the default should really be only high to avoid false positives

/hold

Reply:
severity high or confidence high?

I often see value in the medium ones, that's why this was the suggestion.

Reply:
They're configurable now. thanks for the feedback.

I would still prefer them to be medium. instead of high; any chance to discuss this?

Reply:
my reasoning is to limit false positives (given it is configurable you can choose lower confidence if you want). given the false positives means polluting code with dummy comments - I'l like to see what the difference is - have you tried it against one of our big repositories (or origin before the split)? if this ever gets hooked into CI the confidence should be high